libtsuba/src/RDGPartHeader.cpp

```diff
@@ -191,6 +258,27 @@ tsuba::from_json(const json& j, tsuba::RDGPartHeader& header) {
   j.at(kEdgePropertyKey).get_to(header.edge_prop_info_list_);
   j.at(kPartPropertyFilesKey).get_to(header.part_prop_info_list_);
   j.at(kPartProperyMetaKey).get_to(header.metadata_);
+
+  if (auto it = j.find(kStorageFormatVersionKey); it != j.end()) {
+    it->get_to(header.storage_format_version_);
+  } else {
+    header.storage_format_version_ =
+        RDGPartHeader::kPartitionStorageFormatVersion1;
+  }
+
+  if (header.storage_format_version_ >=
+      RDGPartHeader::kPartitionStorageFormatVersion2) {
+    j.at(kNodeEntityTypeIDArrayPathKey)
```

Comment:
There is a flavor that returns by value:
`template<typename T> T get()`
But it's more since you can't take advantage of type inference like you can in the `get_to` case.